assets/scripts/dialogs/__tests__/GeotagDialog.test.js

```diff
@@ -54,43 +70,66 @@ describe('GeotagDialog', () => {
     expect(wrapper.queryByText('Confirm location')).not.toBeInTheDocument()
   })
 
-  // TODO: Fix these tests
-  it.skip('allows a location to be confirmed when the current signed-in user is the street owner', () => {
+  /*
+  the tests for location pop up already test whether the popup shows up or not
+  tests for StreetMetaGeotag also cover some of this behavior
+  do we have a setup available for mocking a signed-in vs anonymous user? */
+  it('allows a location to be cleared when the current signed-in user is the street owner', () => {
     isOwnedByCurrentUser.mockReturnValueOnce(true)
-    const wrapper = renderWithReduxAndIntl(<GeotagDialog />, { initialState })
+    const wrapper = renderWithReduxAndIntl(<GeotagDialog />, {
+      initialState
+    })
 
-    expect(wrapper.queryByText('Confirm location')).toBeInTheDocument()
+    expect(wrapper.queryByText('Clear location')).toBeInTheDocument()
   })
 
-  it.skip('allows a location to be confirmed when the current anonymous user started this street', () => {
+  /* this test is literally the same as the above test...so again its either redundant or missing something?
+  we can also test more things here, like actually confirming/adding the new location, though thats also
+  covered a little bit by cypress at this point
+  */
+  it.skip('allows a location to be cleared when the current anonymous user started this street', () => {
```

Comment:
You're right, this seems redundant since we have `isOwnedByCurrentUser`, which covers the cases of both a signed-in user (and owner of the street), and anonymous users who do have control over the street. Both cases shouldn't differ and just using that utility function covers both cases. I'd recommend completely deleting this one (keeping it around, but skipped, would only introduce confusion in the future), and updating the description of the above test to just refer to a "street owner," rather than distinguish between a signed-in user or anonymous user.